README.md

```diff
@@ -71,24 +74,45 @@ A modern, enterprise-ready business intelligence web application.
 ## Why Superset
 
 Superset provides:
-* An intuitive interface to explore and visualize datasets, and
-    create interactive dashboards.
-* A wide array of beautiful visualizations to showcase your data.
-* Easy, code-free, user flows to drill down and slice and dice the data
-    underlying exposed dashboards. The dashboards and charts act as a starting
-    point for deeper analysis.
-* A state of the art SQL editor/IDE exposing a rich metadata browser, and
-    an easy workflow to create visualizations out of any result set.
-* An extensible, high granularity security model allowing intricate rules
-    on who can access which product features and datasets.
-    Integration with major
-    authentication backends (database, OpenID, LDAP, OAuth, REMOTE_USER, ...)
-* A lightweight semantic layer, allowing to control how data sources are
-    exposed to the user by defining dimensions and metrics
-* Out of the box support for most SQL-speaking databases
-* Deep integration with Druid allows for Superset to stay blazing fast while
-    slicing and dicing large, realtime datasets
-* Fast loading dashboards with configurable caching
+
+Viewed
+
+A modern, enterprise-ready business intelligence web application.	Apache Superset (Incubating) is a modern,
+enterprise-ready business intelligence web application. It is fast, lightweight, intuitive,
+and loaded with options that make it easy for users of all skill sets to explore and visualize
+their data, from simple pie charts to highly detailed deck.gl geospatial charts.
+
+[**Why Superset**](#why-superset) |
+[**Supported Databases**](#supported-databases) |
+
+## Why Superset
+
+Superset provides:
+- A rich set of data visualizations out of the box
+- An easy-to-use interface for exploring and visualizing data
+- The ability to easily create and share dashboards
+- Enterprise-ready authentication with integration with major authentication providers
+  (database, OpenID, LDAP, OAuth & REMOTE_USER through Flask AppBuilder)
+- An extensible, high-granularity security/permission model allowing intricate rules on who can access
+  individual features and the dataset
+- A simple semantic layer, allowing users to control how data sources are displayed in the UI by defining
+  which fields should show up in which drop-down and which aggregation and function metrics are made available to the user
+- Integration with most SQL-speaking RDBMS through SQLAlchemy
+- Deep integration with Druid.io
+
+Superset is cloud-native and designed to be highly available. It was designed to scale out to large,
```

Comment:
wondering where these README updates are coming from, is that content that Kevin copied/wrote/recommended?